.github/workflows/ci_cd.yml

```diff
@@ -93,6 +93,23 @@ jobs:
               exit 1
           fi
 
+  dependabot-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: dangoslen/dependabot-changelog-helper@v3
+        with:
+          version: 'Unreleased'
```

Comment:
Also, you might not even need to define the version since you are in line with their default

Reply:
Yeah, I was wondering the same thing. I can remove it